src/molecules/Filter/index.js

```diff
@@ -22,17 +22,19 @@ function Filter(props) {
 
   return (
     <div className={classnames(...classes)}>
-      { label &&
+      { label && (
+        /* eslint-disable jsx-a11y/label-has-for */
         <label className={styles['label']} htmlFor={forProp}>
           { label }
         </label>
-      }
+        /* eslint-enable jsx-a11y/label-has-for */
+      )}
 
       <div className={styles['select-wrapper']}>
         <select
           className={styles['select']}
           id={forProp}
-          onChange={onChange}
+          onBlur={onChange}
```

Comment:
why was this updated?

Reply:
`onBlur must be used instead of onchange, unless absolutely necessary and it causes no negative consequences for keyboard only or screen reader users  jsx-a11y/no-onchange`

Reply:
discussed offline - leaving as `onBlur`